terraform/alerting/module.availability-slo/variables.tf

```diff
@@ -1,26 +1,27 @@
-variable "custom-service-id" {
+variable "custom_service_id" {
   type = string
 }
 
-variable "service-name" {
+variable "service_name" {
   type = string
 }
 
-variable "enabled" {
-  type        = bool
-  description = "Whether to enable this availability SLO."
-}
-
 variable "goal" {
   type        = number
   description = "Availability SLO goal."
 }
 
-variable "notification-channels" {
+variable "notification_channels" {
```

Comment:
This will require updates to all the projects in -infra

Reply:
This is a module used by en-alerting only, no need to change the infra repo.